### PR TITLE
Hm 04

### PR DESCRIPTION
**Pull request by a1442456**
hm_04 finished

### CODE REVIEW

File: HomeWorkSolution/Task3/Program.cs [hunk 1]
```diff
@@ -8,29 +8,64 @@ namespace Task3
...
         /// Calculate deposit.
         /// </summary>
-        /// <param name="startDeposit">Start value.</param>
+        /// <param name="currentDeposit">Start value.</param>
         /// <returns>Return deposit value included bank rate.</returns>
-        static void CalculateDeposit(decimal startDeposit, DateTime depositeDate, int monthCount)
+        static void CalculateDeposit(decimal currentDeposit, DateTime depositeDate, uint monthCount, bool isHighPrecision)
         {
+            Console.WriteLine(string.Format("{0} Deposit money at start: {1}; Bank rate: {2}% ", depositeDate.ToShortDateString(), currentDeposit, bankRate));
             decimal addedMoney;
```
Comment (a1442456) [inline HomeWorkSolution/Task3/Program.cs]: In a String.Format() method need to chenge to "Bank rate: {2:P2}" to see % better. Didn't see that before. 
Typing here cz don't want to do new commit.

File: HomeWorkSolution/Task1/Program.cs [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+            }
+            catch (OverflowException ex)
+            {
+                Console.WriteLine(ex.Message);
+                return ReadAnswerFromUser();
+            }
+        }
+    }
+}
+
```
Comment (VadzimPapko) [inline HomeWorkSolution/Task1/Program.cs]: well done

File: HomeWorkSolution/Task2/Program.cs [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+                    Console.WriteLine("-I'm moving DOWN!");
+                    break;
+                case 'a':
+                    Console.WriteLine("-I'm moving LEFT!");
+                    break;
+                case 'd':
+                    Console.WriteLine("-I'm moving RIGHT!");
+                    break;
+                default:
+                    Console.WriteLine("-You type wrong command to the person.\nNext time he'll come to you and show you what to do >_<");
```
Comment (VadzimPapko) [inline HomeWorkSolution/Task2/Program.cs]: you can split this long text message

File: HomeWorkSolution/Task2/Program.cs [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+                    break;
+            }
+        }
+
+        /// <summary>
+        /// Ask user: is moving next?
+        /// </summary>
+        /// <param name="inputedCharacter"></param>
```
Comment (VadzimPapko) [inline HomeWorkSolution/Task2/Program.cs]: param section is not described

File: HomeWorkSolution/Task3/Program.cs [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+                return ReadAnswerFromUser();
+            }
+            catch (OverflowException ex)
+            {
+                Console.WriteLine(ex.Message);
+                return ReadAnswerFromUser();
+            }
+        }
+    }
+}
```
Comment (VadzimPapko) [inline HomeWorkSolution/Task3/Program.cs]: well done!

**Pull request merged**